docker/scenarioo/Dockerfile

```diff
@@ -9,5 +9,7 @@ ENV SCENARIOO_DATA /scenarioo/data
 # ADD SCENARIOO.WAR TO DOCKER IMAGE
 ADD ./scenarioo-latest.war /scenarioo/bin/scenarioo.war
 
+WORKDIR /scnearioo/bin
```

Comment:
Typo: /scenarioo/bin

Reply:
fixed in 369216993decd95ffd46bf8a4006991f02af51a9

Reply:
Thanks!